docs/turnkey-solution/hardware-landscape.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_label: Hardware-Landscape
+sidebar_position: 99
+---
+
+# The SCS Hardware-Landscape
+
+![An image of the SCS hardware landscape rack](images/combined_rack_visual.jpg)
+
+## General information
+
+The general aim of this environment is to install and operate the SCS reference implementation on hardware.
+In addition to the classic tasks in the area of quality assurance, the environment is also used to evaluate
+new concepts in the underlay/overlay network area, as a test environment for hardware-related developments,
+as a demonstration environment for interested parties and as a publicly accessible blueprint for users.
+The environment is designed for long-term use which a varying circle of users.
+
+The environment consists of 21 server and 12 switch components. The selection of hardware and the
+functions and properties used was designed so that the focus is on generally available or characteristic
+functions and dependency on manufacturer-specific functions is avoided. Instead of the x86 servers or SONiC
+switches used here, the realised environment could also be realised with hardware from other manufacturers.
+
+From 1 January 2025, the environment will be operated by [forum SCS-Standards](https://scs.community/2024/10/23/osba-forum-scs-standards/)
+and the participating companies.
```

Comment:
Not really sure why this needs to be part of the documentation. This is content better suited for a blogpost.

Reply:
Thanks, @maxwolfs , @bitkeks  and I decided at the time that it would be a good idea to create a dedicated page for the turnkey solution in the SCS Hardware Landscape variant to show an example of what a turnkey solution installation looks like.